src/Azure.WebSites.DataProtection/Azure.WebSites.DataProtection.csproj

```diff
@@ -36,7 +38,7 @@
 
   <ItemGroup Condition="'$(TargetFramework)' == 'net46'">
```

Comment:
This `ItemGroup` can be deleted, it is already included on line 20

Reply:
This is for net46.